packages/flutter/lib/src/services/font_loader.dart

```diff
@@ -65,6 +65,9 @@ class FontLoader {
     return Future.wait(loadFutures.toList());
   }
 
+  /// Hook called to load a font asset into the engine.
+  ///
+  /// Used to test this object. Not for use in production.
```

Comment:
it's called by `load()`, so presumably it is used in production?

Reply:
This function should not be called by users of this class.  It's effectively private and is only exposed so it can be overridden in tests.

Reply:
updated - PTAL